src/components/page-header/page-header.component.ts

```diff
@@ -7,6 +7,8 @@ import { PageHeaderIconMenu } from './interfaces';
 import { PageHeaderNavigationItem } from './navigation/navigation.component';
 import { PageHeaderNavigation, PageHeaderService } from './page-header.service';
 
+export type NavigationAlignment = 'left' | 'right' | 'center';
```

Comment:
If you are going to export this, it should be named `PageHeaderNavigationAlignment`.